src/index.js

```diff
@@ -41,6 +43,12 @@ export const ActionCreators = {
   },
   redo () {
     return { type: ActionTypes.REDO }
+  },
+  jumpToFuture (index) {
+    return { type: ActionTypes.JUMP_TO_FUTURE, index: index }
```

Comment:
can you use `index` instead of `index: index` here too? (like below, just for consitency)